readme.md

```diff
@@ -258,6 +258,7 @@ Thanks for contributing! 🦋🙌
 - [](# "git-checkout-pr") [Adds copy-pastable git commands to checkout a PR.](https://user-images.githubusercontent.com/16872793/96938908-8e024f80-1499-11eb-8976-0caf95175dd6.png)
 - [](# "jump-to-change-requested-comment") [Adds a link to jump to the latest changed requested comment.](https://user-images.githubusercontent.com/19198931/98718312-418b9f00-23c9-11eb-8da2-dfb616e95eb6.gif)
 - [](# "view-last-pr-deployment") [Adds a link to open the last deployment from the header of a PR.](https://user-images.githubusercontent.com/16872793/111904355-ef185a00-8a1c-11eb-97dd-aa0272547e73.png)
+- [](# "hide-empty-split-diff-side") [Hides one column in the files that don't benefit from split views.](https://user-images.githubusercontent.com/46634000/120284838-faa9bc00-c2bc-11eb-87b2-fce1bcd71f47.png)
```

Comment:
Rather than "hiding", we're just changing view. So it could be "Always uses single-column diffs on files where split diffs aren't useful" or something like that.

Also what's that "View full" button? If it's not native it should not be part of the screenshot.

The name should also reflect this. Suggestions? I don't want to use the word "force" but "force-unified-view" would almost be correct. "no-useless-unified-diff"? Alternatives?

Reply:
`no-useless-split-diff-view` sounds good to me :+1: